test/CMakeLists.txt

```diff
@@ -133,6 +133,12 @@ foreach(file ${files})
     )
     set_tests_properties("${testcase}_all" PROPERTIES LABELS "all")
 
+    # Increase timeout for test-unicode_all on Debug build
```

Comment:
As this only affects a certain configuration at AppVeyor, I would propose to leave the CMakeLists.txt unchanged, but add a parameter to the `ctest` call like `ctest --timeout 3600`.

Reply:
Sure! It just increases the timeout for all unit-tests to 1 hour. Is it OK?

Reply:
Yes - so far, all but the Unicode test are usually fast. (And since this affects AppVeyor only, I do not care the timeout is ever reached.)

Reply:
Done.

**Sidenote:** As `Debug` build for MSVC is now working, it takes a long time for `test-unicode_all` to finish (probably since debug binaries are much slower on MSVC and also AppVeyor _crawls_ to run). I set it to ~30min intially, but even that wasn't enough. Now it is set to 1 hour.